js/src/product-feed/product-feed-table-card/index.js

```diff
@@ -17,12 +17,13 @@ import {
 	Table,
 	TablePlaceholder,
 } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
 import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
-import recordEvent, { recordTablePageEvent } from '.~/utils/recordEvent';
+import { recordTablePageEvent } from '.~/utils/recordEvent';
```

Comment:
❓  Why we are maintaining this `recordTablePageEvent` in the `utils.js` ?

Reply:
It is used in two files `js/src/product-feed/issues-table-card/index.js`, `js/src/product-feed/product-feed-table-card/index.js`, so we need to share it somewhere, so I left it where it was to reduce the number of changes.

Do you have a better idea? Maybe somewhere in `js/src/product-feed/`, but then we would need a dedicated file anyway and I'm not sure if it's worth scattering those functions across more and more files.